packages/core/src/components/sections/ProductDetails/ProductDetails.tsx

```diff
@@ -7,9 +7,11 @@ import {
   DiscountBadge as UIDiscountBadge,
   QuantitySelector as UIQuantitySelector,
   ProductTitle as UIProductTitle,
-  Price as UIPrice,
 } from '@faststore/ui'
 
+import { Components } from './Overrides'
+const { Price: UIPrice } = Components;
```

Comment:
```suggestion
const { Price: UIPrice } = Components
```

Reply:
Also, I wouldn't call it `UIPrice`, it can be confusing since it doesn't always come from faststore/ui.